specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+    <td>URL (string)</td>
+    <td>The resolved address of the <code>spec.subscriber</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>deadLetterSinkUri</td>
+    <td>URL (string)</td>
+    <td>If <code>spec.delivery.deadLetterSink</code> is specified, the resolved URL of the dead letter address.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### Channel v1
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+#### Spec:
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>channelTemplate</code></td>
+    <td>object<br/>(Optional)</td>
+    <td>Implementation-specific parameters to configure the channel.</td>
+    <td>OPTIONAL</td>
+  </tr>
+  <tr>
+    <td><code>subscribers</code></td>
+    <td>[]<a href="#duckv1subscriberspec">duckv1.SubscriberSpec</a></td>
+    <td>Aggregated subscription information; this array should be managed automatically by the controller.</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>delivery</code></td>
+    <td><a href="#deliveryspec">DeliverySpec</a><br/>(Optional)</td>
+    <td>A default delivery options for Subscriptions which do not specify more-specific options.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+#### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>conditions</code></td>
+    <td><a href="#error-signalling">See Error Signalling</a></td>
+    <td>Used for signalling errors, see link. Conditions of types Ready MUST be present.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>observedGeneration</code></td>
+    <td>int64</td>
+    <td>The latest metadata.generation that the reconciler has attempted. If <code>observedGeneration</code> is updated, <code>conditions</code> MUST be updated with current status in the same transaction.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td><a href="#duckv1addressable">duckv1.Addressable</a></td>
+    <td>Address used to deliver events to the Broker.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>subscribers</code></td>
+    <td>[]<a href="#duckv1subscriberstatus">duckv1.SubscriberStatus</a></td>
+    <td>Resolved addresses for the <code>spec.subscribers</code> (subscriptions to this Channel).</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>deadLetterSinkUri</td>
+    <td>URL (string)</td>
+    <td>If <code>spec.delivery.deadLetterSink</code> is specified, the resolved URL of the dead letter address.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### Subscription v1
```

Comment:
We should probably remove the "v1" from these titles since everything in these specs are "v1", right?